api/config/config_stable.json

```diff
@@ -31,6 +32,7 @@
     "projectId": "all-of-us-rw-stable"
   },
   "admin": {
+    "loginUrl": "http:\/\/all-of-us-workbench-stable.appspot.com/login",
```

Comment:
We have a prod config now as well, please add there

Reply:
See above